ui/frontend/build_src/src/components/displayPanel/index.tsx

```diff
@@ -31,17 +32,15 @@ export default function DisplayPanel() {
       return imageData;
     });
 
-    console.log('completedQueries', completedQueries);
-
     if (completedQueries.length > 0) {
       // map the completedImagesto a new array 
       // and then set the state
       const temp = completedQueries.map((query, index ) => {
-        // debugger;
         if(void 0 !== query) {
           //@ts-ignore
           return query.output.map((data)=>{
-            return {id: `${completedIds[index]}-${data.seed}`, data: data.data}
+            // @ts-ignore
+            return {id: `${completedIds[index]}-${data.seed}`, data: data.data, info: {...query.request, seed:data.seed } }
```

Comment:
marrying the individual seed with the id for a react key and with the original request so we have the correct seed for each image in the set.